lib/v2/screens/authentication/recover/recover_account_found/recover_account_found_screen.dart

```diff
@@ -1,20 +1,91 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/components/divider_jungle.dart';
+import 'package:seeds/v2/components/search_result_row.dart';
+import 'package:seeds/v2/components/snack_bar_info.dart';
+import 'package:seeds/v2/components/text_form_field_custom.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/design/app_theme.dart';
 import 'package:seeds/v2/screens/authentication/recover/recover_account_found/interactor/recover_account_found_bloc.dart';
+import 'package:seeds/v2/screens/authentication/recover/recover_account_found/interactor/viewmodels/recover_account_found_events.dart';
 import 'package:seeds/v2/screens/authentication/recover/recover_account_found/interactor/viewmodels/recover_account_found_state.dart';
 
 class RecoverAccountFoundScreen extends StatelessWidget {
   const RecoverAccountFoundScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    List<String>? userGuardians = ModalRoute.of(context)!.settings.arguments as List<String>?;
     return BlocProvider(
-      create: (BuildContext context) => RecoverAccountFoundBloc(),
+      create: (BuildContext context) => RecoverAccountFoundBloc(userGuardians ?? [])..add(FetchInitialData()),
       child: Scaffold(
-        appBar: AppBar(),
+        appBar: AppBar(
+          title: const Text("Recover Account"),
+        ),
```

Comment:
Why you do not like to do the in line when it can be done :p

Reply:
haha i didnt notice, fixed in next PR

Reply:
nvm fixed in this pr